website/docs/r/compute_network.html.markdown

```diff
@@ -95,7 +95,7 @@ The following arguments are supported:
   of this network in the same region as the router. If set to `GLOBAL`,
   this network's cloud routers will advertise routes with all
   subnetworks of this network, across regions.
-  Possible values are `REGIONAL` and `GLOBAL`.
+  Possible values are `REGIONAL` and `GLOBAL`. Defaults to `REGIONAL`.
```

Comment:
This one seems a bit tricky as it looks like the default comes from the API, which typically means the default will change depending on different settings. I'd be hesitant to add the default here.